VMAccess/vmaccess.py

```diff
@@ -95,10 +96,12 @@ def enable():
             _backup_sshd_config(SshdConfigPath)
 
         if reset_ssh:
+            waagent.AddExtensionEvent(name=hutil.get_name(), op="scenario", isSuccess=True, message="reset-ssh")
             _reset_sshd_config(SshdConfigPath)
             hutil.log("Succeeded in reset sshd_config.")
 
         if remove_user:
+            waagent.AddExtensionEvent(name=hutil.get_name(), op="scenario", isSuccess=True, message="remove-user")
```

Comment:
I am more interested in intent as opposed to success.  I can track success (or not) by using the Enable event, and correlating it with the scenario event.